governance.md

```diff
@@ -0,0 +1,101 @@
+
+The Technical Community will require a Leader to drive the development of the Project, including facilitation of Technical Community contributions to Project milestones and of resolution of any Community disagreements of a technical nature, eg relating to the architecture or achievement of technical milestones. The Leader will also represent the overall Technical Community within the Steering Committee. In brief, the Leader should be an excellent project manager and diplomat in addition to possessing technical vision and skills.
+
+Initially, the Project’s technical founder will be the Technical Community Leader for a term of 1 year from the Community adoption of this Governance document. This term is intended to provide for a broad transfer of knowledge and socialization of Project goals and technical norms to the Technical Community as a whole.
+
+Thereafter, Technical Community Leaders (TCLs) will be nominated or can self-nominate from among the existing Committers. The TCL will be elected by a simple majority or greatest plurality for a term of 1 year by a private electronic vote of community members with documented contributions accepted in the previous 12 months (“Voters”). 
+
+The Technical Community will also select a Curriculum Lead according to the same process as Project Leads. The Curriculum Lead is responsible for recruiting and doing the technical onboarding and support of new content developers, in accordance with the Contribution Process and Inbound Code Review Policy. The Curriculum Lead will also maintain a list of curriculum in progress, prioritized by relevance to Scope and Curriculum criteria, and help ensure regular and relevant curriculum releases.
+
+While the Project aims to operate as a consensus-based community, if any Technical Community decision requires a vote to move the Project forward, the eligible voters in the Technical Community will do so by simple majority. Votes may be timebound and held either during meetings or electronically, with the voters’ choices publicly recorded, but at least 50% of eligible Community voters must participate within the specified period of time in order for a vote to take effect. With the exception of changes to Governance or Policy, votes will be decided by a simple majority or greatest plurality (in the case of multiple options). Governance or Policy changes will require approval of ⅔ of the voting quorum.
+
+### Steering Committee
+
+The Steering Committee is responsible for setting the strategic direction of the Project in accordance with its Mission. It provides approval and oversight of the Project’s budget and disbursements, Policy documents and other legal matters, marketing, and other non-technical activities of the Project. Committee business will be conducted in English and will be recorded for public access. 
```

Comment:
Again, recording (as in Youtube videos) seems a bit too strong a requirement : discussing confidential matters like potential sponsors, exclusion of people, etc. Such should probably be conducted in private channels until agreed on, for legal reasons, so I doubt recordings could be made, in practice. I'd suggest to only mandate documentation/minutes of the meetings, allowing to exclude confidential elements from the records.